src/stripeJavaLanguageServer/extractServer.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-sync */
+
+'use strict';
+
+// Import
+const request = require('superagent');
+const fs = require('fs');
+const tar = require('tar-fs');
+const zlib = require('zlib');
+
+// Eclipse JDT source
+// https://www.eclipse.org/community/eclipse_newsletter/2017/may/article4.php
+const href = 'http://download.eclipse.org/jdtls/snapshots/';
+const tarFile = 'jdt-language-server-latest.tar.gz';
+
+const tarPath = `./${tarFile}`;
+const source = `${href}/${tarFile}`;
+
+const extractTo = './out/src/stripeJavaLanguageServer';
```

Comment:
When we publish the extension, it'll ignore the contents of the out folder: https://github.com/stripe/vscode-stripe/blob/master/.vscodeignore

I think that makes sense. Since we are still prototyping, we don't need to publish the java language server stuff out when we make releases. But just noting here that we may want to also extract this out to the `dist` folder when we are ready to release

Reply:
https://jira.corp.stripe.com/browse/DX-6804